pkg/config/autodiscover/deployment_info.go

```diff
@@ -77,6 +79,27 @@ func (deployment *DeploymentResource) GetReplicas() int {
 func (deployment *DeploymentResource) GetLabels() map[string]string {
 	return deployment.Metadata.Labels
 }
+func (deployment *DeploymentResource) GetHpa() configsections.Hpa {
+	template := fmt.Sprintf("go-template='{{ range .items }}{{ if eq .spec.scaleTargetRef.name \"%s\" }}{{.spec.minReplicas}},{{.spec.maxReplicas}},{{.metadata.name}}{{ end }}{{ end }}'", deployment.GetName())
+	ocCmd := fmt.Sprintf("oc get hpa -n %s -o %s", deployment.GetNamespace(), template)
+	out := execCommandOutput(ocCmd)
+	if out != "" {
```

Comment:
A minor suggestion to avoid having two returns.
```
        hpa := configsections.Hpa{}
	if out != "" {
	        out := strings.Split(out, ",")
		hpa.MinReplicas, _ = strconv.Atoi(out[0])
		hpa.MaxReplicas, _ = strconv.Atoi(out[1])
		hpa.HpaName = out[2]
	}
	return hpa
```